include/xrpl/protocol/STTx.h

```diff
@@ -120,10 +120,16 @@ class STTx final : public STObject, public CountedObject<STTx>
         @return `true` if valid signature. If invalid, the error message string.
     */
     enum class RequireFullyCanonicalSig : bool { no, yes };
+
     Expected<void, std::string>
     checkSign(RequireFullyCanonicalSig requireCanonicalSig, Rules const& rules)
         const;
 
+    Expected<void, std::string>
+    checkBatchSign(
```

Comment:
Why add these to `STTx` instead of having the code in `Batch.cpp`?

Reply:
Because they were so similar to the other functionality. I can move them. Is this preferred?